cmake/llext-edk.cmake

```diff
@@ -158,13 +161,25 @@ yaml_get(INTERFACE_INCLUDE_DIRECTORIES NAME build_info KEY cmake llext-edk inclu
 yaml_get(APPLICATION_SOURCE_DIR NAME build_info KEY cmake application source-dir)
 yaml_get(WEST_TOPDIR NAME build_info KEY west topdir)
 
+yaml_get(board_name NAME build_info KEY cmake board name)
+yaml_get(board_qualifiers NAME build_info KEY cmake board qualifiers)
+yaml_get(board_revision NAME build_info KEY cmake board revision)
+
+set(board_target ${board_name})
+if(board_qualifiers)
+    set(board_target "${board_target}_${board_qualifiers}")
```

Comment:
technically you're not creating the board target here, because a board target has the form:
![image](https://github.com/user-attachments/assets/f98097f1-6fb7-4132-98e2-7598806bc03c)

https://docs.zephyrproject.org/latest/hardware/porting/board_porting.html#board-terminology

Please use another variable name to avoid confusing readers in the future who looks at this code.